http_Headscale_tutorial.md

```diff
@@ -0,0 +1,99 @@
+# Setting Up a Simple Headscale Control Server Using HTTP
```

Comment:
I dont think this is the place for this documentation, and we have our own documentation. This looks like internal notes.

Reply:
You are correct the has been reverted